components/Cards/ProfileCard.js

```diff
@@ -27,19 +48,68 @@ export default function ProfileCard({
         <StyledLink href={`/profile/community/${foundUser._id}`}>
           {foundUser?.userName || foundUser?.firstName}
         </StyledLink>
-        <FollowButton
-          foundUser={foundUser}
-          user={user}
-          allUsers={allUsers}
-          mutateAllUsers={mutateAllUsers}
-        />
+        {followButton && (
+          <FollowButton
+            foundUser={foundUser}
+            user={user}
+            allUsers={allUsers}
+            mutateAllUsers={mutateAllUsers}
+            mutateUser={mutateUser}
+          />
+        )}
+        {isHousehold && !isEditingHousehold && (
+          <P>({getLabelForMemberRole(member.role)})</P>
+        )}
+        {isEditingHousehold && member.role !== "owner" && (
+          <Wrapper>
+            {" "}
+            {member.role === "canWrite" ? (
+              <Button
+                $height
+                $top
+                onClick={(event) => {
+                  event.preventDefault();
+                  changeMemberRole(member._id, "canRead");
+                  setIsEditingHousehold(false);
+                }}
+              >
+                Schreibzugriff entfernen
+              </Button>
+            ) : (
+              <Button
+                $height
+                $top
+                onClick={(event) => {
+                  event.preventDefault();
+                  changeMemberRole(member._id, "canWrite");
+                  setIsEditingHousehold(false);
+                }}
+              >
+                Schreibzugriff geben
+              </Button>
```

Comment:
It looks like on a couple of things change based on the value of `member.role.` We could potentially reduce the amount of code as follows. 

```suggestion
<Button
  $height
  $top
  onClick={event => {
    event.preventDefault();
    changeMemberRole(member._id, member.role === "canWrite" ? "canRead" : "canWrite");
    setIsEditingHousehold(false);
  }}
>
  {member.role === "canWrite" ? "Schreibzugriff entfernen" : "Schreibzugriff geben"}
</Button>
```